lib/src/css_mirroring/cssjanus_runner.dart

```diff
@@ -0,0 +1,30 @@
+library scissors.src.css_mirroring.cssjanus_runner;
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import '../utils/io_utils.dart';
+
+/// Executes cssjanus(https://github.com/cegov/wiki/tree/master/maintenance/cssjanus).
+/// Input: [source] css and [cssJanusPath] which points to an executable.
+/// Pipes in the source css to cssjanus.
+/// Output: css flipped from ltr to rtl orientation and vice-versa.
```

Comment:
Nit on comment style, suggestion:

/// Runs cssjanus (https://github.com/cegov/wiki/tree/master/maintenance/cssjanus) on
/// [source], and returns the mirrored css.
///
/// [cssJanusPath] points to an executable.